src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
     }
 };
 
+groupTypes.horizBrace = function(group, options) {
+    const style = options.style;
+    const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+        style.sizeMultiplier;
+
+    const haveNote = (group.type === "supsub");
+    let note;
+    let newOptions;
+    let noteReset;
+    if (haveNote) {
+        if (group.value.sup) {
+            newOptions = options.withStyle(style.sup());
+            note = buildGroup(group.value.sup, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sup().cls()],
+                [note], newOptions);
+        } else {
+            newOptions = options.withStyle(style.sub());
+            note = buildGroup(group.value.sub, newOptions);
+            noteReset =  makeSpan([style.reset(), style.sub().cls()],
+                [note], newOptions);
+        }
+        group = group.value.base;
+    }
+
+    // Build the base group
+    const body = buildGroup(
+       group.value.base, options.withStyle(style.cramp()));
+
+    // If a custom bracket, convert units.
+    if (group.value.thickness) {
+        group.value.thickness.value = calculateSize(group.value.thickness.value,
+                options.style);
+    }
+    if (group.value.height) {
+        group.value.height = calculateSize(group.value.height.value,
+                options.style);
+    }
+
+    // Create the stretchy element
+    const braceBody = stretchy.svgSpan(group, options);
+
+    // Generate the vlist, with the appropriate kerns
+    const braceShift = (group.value.isOver ? -body.height - 3 * ruleWidth :
+                        body.depth + braceBody.height + 3 * ruleWidth);
```

Comment:
Yes. I would also like to point out that the new accents include harpoons. They are like overline in that they need a healthy kern distance. Otherwise, a long flat line looks like it is pasted to the top of the subject matter